lib/chunky_png/canvas/resampling.rb

```diff
@@ -139,9 +139,25 @@ def resample_bilinear!(new_width, new_height)
       def resample_bilinear(new_width, new_height)
         dup.resample_bilinear!(new_width, new_height)
       end
+
+      def resize_to_fit!(new_width, new_height)
```

Comment:
This method is missing yardoc comments

Reply:
done -- note all the existing comments describe the non-bang version so I followed your lead.